lib/js/components/Well/Well.consts.ts

```diff
@@ -1,4 +1,13 @@
+import { Value } from '../../utils/type.utils';
+
 export const WELL_PADDINGS = {
 	SMALL: 'small',
 	MEDIUM: 'medium',
 };
+
+export type WellPadding = Value<typeof WELL_PADDINGS>;
+
+export const CHIP_AVAILABLE_PADDINGS = {
+	SMALL: 'small',
```

Comment:
I wonder if this should be coupled with `WELL_PADDINGS`, now it's not, but at the end of the day values in this const are the same values as in `WELL_PADDINGS`.

Reply:
This is used in the "stories" only so yeah, I think we can drop it and just "combine" the array in there from the values of WellPaddings .